.github/.workflows/sourcery.yml

```
@@ -0,0 +1,21 @@
+      - uses: actions/checkout@v3
+        with:
+          # This is necessary because Sourcery needs to compare with the base branch
+          fetch-depth: 0 
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+      - uses: sourcery-ai/action@v1
+        with:
+          token: ${{ secrets.SOURCERY_TOKEN }}
+          # Tell Sourcery to only review the changes in the pull request
+          diff_ref: ${{ github.event.push.base.sha }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)